src/webui/www/private/views/preferences.html

```diff
@@ -969,6 +969,14 @@
                     <input type="checkbox" id="reannounceWhenAddressChanged" />
                 </td>
             </tr>
+            <tr>
+                <td>
+                    <label for="refreshInterval">QBT_TR(Transfer list refresh interval:)QBT_TR[CONTEXT=OptionsDialog]</label>
```

Comment:
It still seems to me that WebUI users will be misled by the wording "Transfer list refresh interval".

Reply:
Do you have any suggestions?

Reply:
>It still seems to me that WebUI users will be misled by the wording "Transfer list refresh interval".

GUI use that wording too.

Reply:
> > It still seems to me that WebUI users will be misled by the wording "Transfer list refresh interval".
> 
> GUI use that wording too...

At least GUI Transfer list refreshes depending on this interval.

Reply:
Should it be renamed to GUI transfer list refresh interval then? xD

Reply:
> Should it be renamed to GUI transfer list refresh interval then? xD

No, of course. Why WebUI should affect GUI settings?..
IMO, it should reflect its core-level nature. "torrent states refresh interval" or something like that.

Reply:
> I’m more inclined towards keeping the settings name same since some users may be using both the GUI and WebUI and they might get confused by the different naming.

Then I'd better change it in GUI too.
However, if the others do not agree with me, you can continue as you wish.

Reply:
Maybe just call it "Refresh interval" ambiguously in both GUI and WebUI? And additionally add a tooltip that states "It controls the internal state update interval which in turn will affect UI updates".